test/driver/index.ts

```diff
@@ -0,0 +1,5 @@
+import './issue-105';
+import './mockDomSource';
+import './MotorcycleDomSource';
+import './vNodeWrapper';
+import './integration';
```

Comment:
No new line EOF. This is needed because of Git diffs. You should set this as a standard so you need not to think about it.